content/en/getting_started/integrations/_index.md

```diff
@@ -68,6 +68,7 @@ logs:
       path: /var/log/apache2/error.log
       source: apache
       sourcecategory: http_web_access
+      service: apache
```

Comment:
this should not be here under logs but under the instance section. Otherwise its not unified

Reply:
yup, or `init_config`

Reply:
➕ 
Just to clarify, to have it really unified in our docs, we should define it under `init_config`, i.e.:

```yaml
init_config:
  service: apache
```

and remove all occurences of `service: apache` from the `logs` section.

Ideally, we could add a comment about `service` being supported under `init_config` only since Agent v7.18/v6.18